README.md

```diff
@@ -1,112 +1,204 @@
+</tr>
+
+<tr>
+  <th>v7</th>
+  <td>Unreleased</td>
+  <td colspan="3" align="center">N/A</td>
 </tr>
+
 </table>
 
 <p><img src="schedule.png" alt="LTS Schedule"/></p>
 
-# LTS Plan
+# Node.js Release and LTS Plan
+
+## Summary
+
+The new Node.js Release and LTS plan took effect in 2015 with the release of Node.js v4.
+While it also takes into account older versions of Node.js, it is easiest to understand when looking forward from v4:
+
+<p><img src="schedule_summary.gif" alt="LTS Schedule Summary"/></p>
+
+## Versioning
+
+Node.js primarily uses [Semantic Versioning](http://semver.org/) ("semver") to identify releases.
+This versioning scheme uses three numbers, separated by a period (".").
+The first of these numbers is the _Major_ version number (we generally refer to this as "semver-major").
+Semver dictates that all incremental releases sharing the same _Major_ number shall be backward compatible, with no intentionally breaking changes introduced throughout this series of releases.
+If you are unfamiliar with Semantic Versioning, please review the documentation available at http://semver.org as Node.js, and the majority of its [package ecosystem](https://npmjs.org), make heavy use of this scheme to identify the types of changes introduced in each release.
+
+## Major releases
+
+New semver-major release lines of Node.js are cut from the `master` development branch every six months.
+New even-numbered release lines (e.g. v6, v8, v10, etc) are cut in April.
+New odd-numbered release lines (e.g. v5, v7, v9) are cut in October.
+These 6-monthly release lines are initially referred to as "**Current**" and receive most non-breaking changes that occur on the `master` development branch of Node.js, hence their semver-minor version numbers can increase rapidly during this period.
+
+## Current ⇒ LTS
+
+In October of each year, the even-numbered "Current" release line is transitioned on to the Long Term Support (LTS) plan and begin to be referred to as "**LTS**".
+At the same time, a new odd-numbered "Current" release line is created.
+
+## Support timeframes
+
+Every release line covered by LTS will be actively maintained for an initial period of 18 months from the date it enters LTS coverage; this is "**Active LTS**".
+Following those 18 months of active support, the release line will transition into "**Maintenance**" mode for 12 additional months.
+
+Given this schedule, there will be no more than two "Active LTS" release lines at any given time, overlapping for a maximum period of six months.
+Users are encouraged to use this overlap to plan a migration path between LTS release lines.
+
+Release lines not covered by LTS (i.e. odd-numbered release lines) receive an additional 2 months of "Maintenance"-style support following their 6 months as "Current" releases.
+
+## Acceptable changes for release lines
+
+All release lines covering a single semver-major version number, be it labelled "Current", "Active LTS" or "Maintenance" will:
```

Comment:
Maintenance branches should never receive minors except for major security issues.